### PR TITLE
Update README to detail local customizations

### PR DESCRIPTION
**Pull request by awoods**
Resolves: https://jira.duraspace.org/browse/FCREPO-2007

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,6 +70,29 @@ will cause the script to pause for user input after each restore operation.
...
+The following services have been configured with Fedora credentials:
+* Solr indexer
+* Triplestore indexer
+* Fixity service
+* Serialization service
+* Reindexing service
+
+The Solr indexer has been configured to communicate with Solr running on port 8080, instead of the default 8983.
+
+The triplestore indexer has been configurated to NOT include any `Prefer` headers. The production default is to limit the triples to be indexed by omitting `ldp:contains` triples.
```
Comment (ajs6f) [inline README.md]: "configurated"?

File: README.md [hunk 2]
```diff
@@ -73,6 +70,29 @@ will cause the script to pause for user input after each restore operation.
 ```
 ./restore_runner.sh
 ``` 
+## Customizations
+
+The applications installed on this Vagrant box have been customized to support showcasing features. Some of the configuration of these applications may not be recommended for production installations.
```
Comment (ajs6f) [inline README.md]: Just "to showcase features".

**Comment by ajs6f** [comment]
Two small changes, otherwise good.

**Comment by awoods** [comment]
Updated. Ready for squash/merge.

**Pull request merged**